LML.NPOManagement.Bll/Services/TokenCreationHelper.cs

```diff
@@ -47,7 +47,7 @@ public static string GenerateJwtToken(UserModel user, IConfiguration configurati
                     Subject = new ClaimsIdentity(new[] {
                     new Claim("Id", user.Id.ToString()),
                 }),
-                    Expires = DateTime.UtcNow.AddMinutes(Convert.ToInt16(configuration.GetSection("AppSettings:TokenExpiration").Value)),
+                    Expires = DateTime.UtcNow.AddMinutes(expire > 0 ? expire : Convert.ToInt16(configuration.GetSection("AppSettings:TokenExpiration").Value)),
                     SigningCredentials = new SigningCredentials(GetSigningKey(configuration), SecurityAlgorithms.HmacSha256Signature)
```

Comment:
I understand why you are sending different expiration timouts to the token generation. Let's configure those parameters and you put a clear key to the app.settings definition so we know what purpose they serve. MAybe we'll want to play with token expiration timeouts for those use cases.